### ISSUE TITLE
Python 3.12?

### ISSUE THREAD

**Issue by mattwthompson** [conda-forge/lammps-feedstock#189]
### Comment:

Hi, thanks for your work on packaging LAMMPS, it's a huge help for the community.

I'm looking at adding LAMMPS to some of my build matrices and I'm not seeing it solvable with 3.12. I didn't see another issue raised here. It's not obvious to me from skimming the [release notes](https://github.com/lammps/lammps/releases) of recent versions if it's supported upstream or not. ~~I can't find~~ nevermind, through [the migrations page](https://conda-forge.org/status/migration/python312) I see this is probably blocked by https://github.com/conda-forge/quippy-feedstock/pull/27?

**Comment by jan-janssen** [comment]
> nevermind, through [the migrations page](https://conda-forge.org/status/migration/python312) I see this is probably blocked by [conda-forge/quippy-feedstock#27](https://github.com/conda-forge/quippy-feedstock/pull/27)?

That's correct. I have not found the time to patch quippy yet. The alternative would be to just drop it for python 3.12 a beyond until they get their code base ready.

**Comment by jan-janssen** [comment]
I dropped quippy for Python 3.12 compatibility in https://github.com/conda-forge/lammps-feedstock/pull/191
